src/book/03-guides/01-main-codebase/02-script-tuner.mdx

```diff
@@ -0,0 +1,99 @@
+
+The next section is a table. These show the angle and gain for each servo for that frame.
+
+The bottom of the interface shows what commands are available. It does not show all the commands. All commands can be viewed by typing `:help`. Note that the letters are UPPERCASE.
+
+| Command                        | Description                                                                                                                                                                                                                                                                                                                                                                                          |
+| ------------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| <kbd>,</kbd>                   | Moves to the frame that is left of the frame you have currently selected. When you move to the frame, the robot's joints will move to the joint positions for that frame.                                                                                                                                                                                                                            |
+| <kbd>.</kbd>                   | Moves to the frame that is right of the frame you have currently selected. When you move to the frame, the robot's joints will move to the joint positions for that frame.                                                                                                                                                                                                                           |
+| <kbd>N</kbd>                   | Creates a new frame after the frame you have currently selected. This frame will be a duplicate of the frame you have selected. It will be added in after the currently selected frame, and the selected frame will not change.                                                                                                                                                                      |
+| <kbd>I</kbd>                   | Deletes the current frame you have selected. The new selected frame will be the frame after the deleted frame, or before if it was the last frame.                                                                                                                                                                                                                                                   |
```

Comment:
```suggestion
| <kbd>I</kbd>                   | Deletes the currently selected frame. The frame after the deleted frame will be selected, if there's one. Otherwise the one before will be selected.                                                                                                                                                                                                                                                   |
```